README.md

```diff
@@ -27,8 +27,8 @@ Please see [our Showcase page](https://weareadaptive.com/showcase/) for a full l
 [finsemble]: https://cosaic.io/finsemble/
 [storybook]: https://www.reactivetrader.com/storybook
 [style guide]: https://www.reactivetrader.com/styleguide
-[openfin-win]: ./install/Reactive-Launcher-Demo.exe?raw=true
```

Comment:
I've half a mind to remove these .. and just point to the install README ..

Reply:
Yes, done